asab/storage/__init__.py

```diff
@@ -12,7 +12,8 @@ class Module(asab.Module):
 
 	def __init__(self, app):
 		super().__init__(app)
-		sttype = asab.Config.get('asab:storage', 'type')
+		# old configuration format [asab:storage]
+		sttype = asab.Config.get('asab:storage', 'type', fallback=None)
```

Comment:
`None` is the default, you don't have to put it here explicitly, am I right?

Reply:
Nope, I get an error without it.

Reply:
Alright.